app/src/main/java/ru/prometeydev/movie/FragmentMoviesDetails.kt

```diff
@@ -44,32 +44,38 @@ class FragmentMoviesDetails : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         loadData()
 
-        view.findViewById<ImageView>(R.id.movie_logo)
-            .setImageResource(movie?.additional!!.pictureDrawable)
+        movie?.let {
+            view.findViewById<ImageView>(R.id.movie_logo)
+                    .load(it.backdrop)
 
-        view.findViewById<TextView>(R.id.age_limit)
-            .text = context?.getString(R.string.age_limit, movie?.ageLimit)
+            view.findViewById<TextView>(R.id.age_limit)
+                    .text = getString(R.string.age_limit, it.minimumAge)
```

Comment:
Minor - `it` is fine for one liner and it is harder to follow when it is used in many places. Consider to have something meaningful

Reply:
Improve it